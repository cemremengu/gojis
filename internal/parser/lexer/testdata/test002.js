console.log("some string parameter");