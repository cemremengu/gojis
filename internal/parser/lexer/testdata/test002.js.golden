IdentifierName
Dot
IdentifierName
ParOpen
StringLiteral
ParClose
SemiColon
